apps/web/src/pages/Pools/PoolAdvertisementPage/PoolAdvertisementPage.tsx

```diff
@@ -529,7 +547,12 @@ export const PoolPoster = ({
                 isIAP={pool.publishingGroup === PublishingGroup.Iap}
                 classificationGroup={classificationGroup}
               />
-              <Heading level="h3" size="h4">
+              <Heading
+                level="h3"
+                size="h4"
+                data-h2-font-weight="base(700)"
```

Comment:
Same train of thought as the Separator - but yes, 100% agree. Eric and I have already started talking about this one and intend to bring Brooke into the convo so that we can establish a consistent, reusable hierarchy.